client/src/components/profileInfoEdit/ProfileInfoEdit.js

```diff
@@ -61,25 +50,57 @@ function ProfileInfoEdit () {
                              <Form.Control type="text" placeholder="+62123456789" />
                         </Form.Group>
 
+                        <Form.Group controlId="formWebUrl">
+                             <Form.Label>Website URL</Form.Label>
+                             <InputGroup className="mb-3">
+                                <Form.Control 
+                                    type="text"
+                                    placeholder="Team custom URL"
+                                    aria-label="Team custom URL"
+                                    aria-describedby="basic-addon2"
+                                />
+                                <InputGroup.Append>
+                                    <InputGroup.Text id="basic-addon2">researchify.com</InputGroup.Text>
+                                </InputGroup.Append>
+                             </InputGroup>
+                        </Form.Group>
+
+                        <div className= "my-1">
+                            <Button 
+                                color="primary"
+                                className= "mr-2"
+                                onClick={(event) => {
+                                    event.preventDefault();
+                                    alert('You have updated your profile')
+                                    /**
+                                     * Update user information here onclick button
+                                     */
```

Comment:
I know this isn't complete yet, but just putting it out there when we do implement this, if we're thinking of keeping this `alert`, I think we could instead use something like a `Toast` instead. Here's a really simple, beautiful and my favourite Toast library that I've introduce in `thunder`: https://react-hot-toast.com, feel free to have a look :)

Reply:
Changed alert to toast, thanks for suggestion